components/footer/footer-link.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import useTheme from '../use-theme'
+
+interface Props {
+  target?: React.AnchorHTMLAttributes<HTMLAnchorElement>['target']
+  href: string
+}
+
+type NativeAttrs = Omit<React.HTMLAttributes<any>, keyof Props>
+export type FooterLinkProps = Props & NativeAttrs
+
+const FooterLink: React.FC<React.PropsWithChildren<FooterLinkProps>> = ({
+  href,
+  children,
+  target,
+  ...props
+}) => {
+  const theme = useTheme()
+  return (
+    <li {...props}>
+      <a href={href} target={target || '_self'}>
+        {children}
+      </a>
+      <style jsx>{`
+        li:before {
+          content: '' !important;
+        }
+        li :global(a) {
+          color: ${theme.palette.accents_5};
+          transition: color 0.1s ease;
+        }
+        li :global(a):hover {
```

Comment:
If you are disturbed by a lot of `style` overrides, you can use `div` instead of `li`. (you can use the `area-*` attribute to ensure semantics.)

Reply:
I think `li` does the job here